third_party/go-tun2socks/METADATA

```diff
@@ -15,6 +15,6 @@ third_party {
   local_modifications:
     "outline-go-tun2socks consumes go-tun2socks in order to build libraries "
     "and binaries for Outline platforms."
-    "Temporarily updated Android clients to libraries built at commit 5d427b6, "
+    "Temporarily updated Apple clients to libraries built at commit bd58355, "
```

Comment:
Is the statement about Android no longer true?

Reply:
It is, this fixes a typo (see the next line).